Conversions/DateToDay.js

```diff
@@ -38,25 +19,23 @@ const daysNameList = { // weeks-day
   5: 'Friday',
   6: 'Saturday'
 }
-
 const DateToDay = (date) => {
-  // firstly, check that input is a string or not.
+ // firstly, check that input is a string or not.    
   if (typeof date !== 'string') {
     return new TypeError('Argument is not a string.')
   }
-  // extract the date
-  const [day, month, year] = date.split('/').map((x) => Number(x))
+  //   // extract the date
+  let [day, month, year] = date.split('/').map((x) => Number(x))
   // check the data are valid or not.
   if (day < 0 || day > 31 || month > 12 || month < 0) {
     return new TypeError('Date is not valid.')
   }
-  // divide year to century and yearDigit value.
-  const yearDigit = (year % 100)
-  const century = Math.floor(year / 100)
-  // Apply the algorithm shown above
-  const weekDay = Math.abs((day + Math.floor((2.6 * calcMonthList[month]) - 0.2) - (2 * century) + yearDigit + Math.floor(yearDigit / 4) + Math.floor(century / 4)) % 7)
-  // return the weekDay name.
-  return daysNameList[weekDay]
+  // date is resolved based on Zeller's congruence.
+  if (month < 3) { --year; month += 12; }
```

Comment:
I'd use `year--` here

Reply:
I will make the change also can you explain me why post decrement would be better in this case ? Just curious.

Reply:
I have changed it to year-- should I make another pull request?